runtime/oti/util_api.h

```diff
@@ -1310,11 +1310,14 @@ isObjectStackAllocated(J9VMThread *targetThread, j9object_t aObj);
 
 /**
 * @brief
-* @param *javaVM
-* @param *library
-* @return void
+* @param *javaVM the J9JavaVM.
+* @param *libraryName a null terminated string containing the library name.
+* @param *libraryHandle the handle of a JNI library providing the libName.
```

Comment:
I think `libName` should be `libraryName` here.

Reply:
Updated